static/styles/compose.css

```diff
@@ -422,6 +422,10 @@ a.message-control-button {
     }
 }
 
+a.message-control-button.hide {
+    display: none;
+}
```

Comment:
I guess there may be a better way to do this once we change compose to a handlebars template.